phpcs.xml.dist

```diff
@@ -14,6 +14,10 @@
         <!-- Special test where there are multiple classes in a file -->
         <exclude-pattern>tests/test_files/ClassC.php</exclude-pattern>
     </rule>
+    <rule ref="PEAR.Files.IncludingFile.UseIncludeOnce">
```

Comment:
this check should be disabled completely

Reply:
That rule is part of the Hostnet custom rules, so disabling it completely is not possible. This is the best solution for now.